clients/flash/air-client/src/Main-app.xml

```diff
@@ -1,30 +1,30 @@
 <?xml version="1.0" encoding="utf-8" standalone="no"?>
-<application xmlns="http://ns.adobe.com/air/application/3.2">
-
-	<!-- Adobe AIR Application Descriptor File Template.
-
-		Specifies parameters for identifying, installing, and launching AIR applications.
-
-		xmlns - The Adobe AIR namespace: http://ns.adobe.com/air/application/3.1
-				The last segment of the namespace specifies the version
-				of the AIR runtime required for this application to run.
-
-		minimumPatchLevel - The minimum patch level of the AIR runtime required to run
-				the application. Optional.
+<application xmlns="http://ns.adobe.com/air/application/23.0">

```

Comment:
The only change in this file appears to be this version change, but there are a bunch of other diffs that don't appear to change anything.